src/main/osd/osd.h

```diff
@@ -222,6 +224,7 @@ typedef struct osdConfig_s {
     uint16_t cap_alarm;
     uint16_t alt_alarm;
     uint8_t rssi_alarm;
+    uint8_t link_quality_alarm;
```

Comment:
This has to go at the end of the parameter group, or else it will break it: https://github.com/betaflight/betaflight/blob/master/docs/development/ParameterGroups.md